frontend/src/metabase/visualizations/visualizations/CartesianChart/events.ts

```diff
@@ -647,6 +655,77 @@ export const getStackedTooltipModel = (
   };
 };
 
+export const getOtherSeriesTooltipModel = (
+  chartModel: BaseCartesianChartModel,
+  settings: ComputedVisualizationSettings,
+  dataIndex: number,
+  datum: Datum,
+) => {
+  const { groupedSeriesModels = [] } = chartModel;
+
+  const rows = groupedSeriesModels
+    .map(seriesModel => ({
+      name: seriesModel.name,
+      column: seriesModel.column,
+      value: datum[seriesModel.dataKey],
+      prevValue: computeDiffWithPreviousPeriod(
+        chartModel,
+        seriesModel,
+        dataIndex,
+      ),
+    }))
+    .sort((a, b) => {
+      if (typeof a.value === "number" && typeof b.value === "number") {
+        return b.value - a.value;
+      }
+      return a.value === undefined ? 1 : -1;
+    })
+    .map(row => ({
+      name: row.name,
+      values: [
+        formatValueForTooltip({
+          value: row.value,
+          column: row.column,
+          isAlreadyScaled: true,
+          settings,
+        }),
+        row.prevValue,
+      ],
+    }));
+
+  return {
+    header: String(
+      formatValueForTooltip({
+        value: datum[X_AXIS_DATA_KEY],
+        column: chartModel.dimensionModel.column,
+        settings,
+      }),
+    ),
+    rows,
+    footer: {
```

Comment:
When there is only one element the footer seems excessive

Reply:
Made it a last row instead [`fc13961` (#48265)](https://github.com/metabase/metabase/pull/48265/commits/fc139619006a8d378a468b5cfa529f8982d6ae25)

![tooltips](https://github.com/user-attachments/assets/15c97777-4750-4397-9cfa-5969d7a81846)